packages/suite/src/components/suite/graph/TransactionsGraph/GraphTooltipBase.tsx

```diff
@@ -200,8 +194,7 @@ export const GraphTooltipBase = (props: GraphTooltipBaseProps) => {
                     {props.balance && (
                         <Row>
                             <Value>
-                                <Sign $color="transparent">+</Sign>
-                                {props.balance}
+                                <Row $noBottomMargin>{props.balance}</Row>
```

Comment:
I suggest to use `Row` component from DS here

Reply:
used